Solutions/SlashNext SIEM/Playbook/deploy.json

```diff
@@ -0,0 +1,1192 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "PlayBook Name": {
+            "defaultValue": "SlashNextSecurityEventsforMicrosoftSentinel",
+            "type": "String"
+        },
+        "Storage Account Name": {
+            "defaultValue": "snxstorage",
+            "type": "String",
+            "metadata": {
+                "description": "Name of Storage account to be created."
+            }
+        },
+        "Process x weeks Data": {
+            "metadata": {
+                "description": "Previous week from which scanning should start."
+            },
+            "type": "int",
+            "defaultValue": 5
+        },
+        "Workspace ID": {
+            "type": "String",
+            "metadata": {
+                "description": "Log analytic workspace ID"
+            }
+        },
+        "Workspace Name": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of log analyrtic workspace."
```

Comment:
spell mistake : analyrtic

Requesting to check whole playbook for spelling mistakes .